src/worker/browser-env/runner/test-runner/index.ts

```diff
@@ -127,3 +179,26 @@ export class TestRunner extends NodejsEnvTestRunner {
         ]);
     }
 }
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+function transformExpectArgs(arg: any): unknown {
+    if (
+        typeof arg === "object" &&
+        "$$typeof" in arg &&
+        Object.keys(SUPPORTED_ASYMMETRIC_MATCHER).includes(arg.$$typeof)
```

Comment:
If `SUPPORTED_ASYMMETRIC_MATCHER` does not includes `arg.$$typeof` (which should mean "asymmetric matcher is not supported") you are just returning the argument raw.

Reply:
here all arguments are processed by this function (not only assymetric matcher). So in some cases argument must be return as is.

Found this solution in wdio - https://github.com/webdriverio/webdriverio/blob/v8.36.0/packages/wdio-runner/src/utils.ts#L200-L213